src/collection/dimensions.js

```diff
@@ -266,6 +266,33 @@ fn = elesfn = ({
         };
       }
 
+      function computePaddingValues( width, height, paddingObject, relativeTo ) {
+        // Assuming percentage is number from 0 to 1
```

Comment:
When a percentage value is parsed, the % sign is removed and the resulting number is divided by 100. This parsed value is retrieved by `ele.pstyle( 'padding' )`. Technically if you put in 1000%, it will still work though, you will just get a padding larger than the width or whatever, so maybe I should remove that comment.